src/main/js/wiki/containers/Settings.tsx

```diff
@@ -74,11 +80,40 @@ const Settings: FC<Props> = (props) => {
           onClick={() => {
             editWikiMutation.mutate({ landingPage: landingPage, rootDir: rootDir });
           }}
+          disabled={
+            !settingsValid({
+              landingPage: landingPage,
+              rootDir: rootDir
+            })
+          }
         />
         <ActionButton i18nKey="settings_abort" onClick={changeToWikiRoot} />
       </div>
     </div>
   );
 };
 
+/**
+ * Checks if the entered configuration is valid.
+ */
+function settingsValid(configuration: Configuration) {
+  const rootDirValid = isValid(configuration.rootDir);
+  const landingPageValid = isValid(configuration.landingPage);
+  return rootDirValid && landingPageValid;
+}
+
+/**
+ * Validates that the given path is a valid path.
+ * Also validates that the path does not start with '.' or '/'
+ * @param path The path to check
+ */
+function isValid(path: string): boolean {
+  let result = isValidPath(path);
+
+  result = result && !path.startsWith("/");
```

Comment:
This should move to the isValidPath. Is there any reason to create a second isValid method.

Reply:
Then this method can be completely replaced by isValidPath